lib/screens/profile_screens/edit_name/interactor/viewmodels/edit_name_bloc.freezed.dart

```diff
@@ -0,0 +1,485 @@
+// coverage:ignore-file
+// GENERATED CODE - DO NOT MODIFY BY HAND
+// ignore_for_file: unused_element, deprecated_member_use, deprecated_member_use_from_same_package, use_function_type_syntax_for_parameters, unnecessary_const, avoid_init_to_null, invalid_override_different_default_values_named, prefer_expression_function_bodies, annotate_overrides, invalid_annotation_target
+
+part of 'edit_name_bloc.dart';
```

Comment:
I am gonna have to teach the `danger` script to ignore generated classes towards the 500 line max

Reply:
no need to review this file, its generated code

Reply:
Omg I know its generated but its impresive the size 817 lines for 3 events